docs/docs/user-guide/training-data-format.mdx

```diff
@@ -0,0 +1,659 @@
+
+If you are using the regular expressions for the `RegexFeaturizer` the
+name of the regular expression does not matter. If does not define the
+entity nor the intent, it is just a human readable description for you
+to remember what this regex is used for and is the title of the
+corresponding pattern feature.
+
+If you want to use the RegexFeaturizer you can also use the regex
+features to improve the intent classification performance, for example,
+by defining a greet clause:
+
+```yaml
+- regex: greet
+  examples: |
+  - hey[^\\s]*
+```
+
+Try to create your regular expressions in a way that they match as few
+words as possible. E.g. using `hey[^\\s]*` instead of `hey.*`, as the
+later one might match the whole message whereas the first one only
+matches a single word.
+
+When using the `RegexFeaturizer`, the regex features for entity extraction
+are currently only supported by the `CRFEntityExtractor` and the
+`DIETClassifier` component! Hence, other entity extractors, like
+`MitieEntityExtractor` or `SpacyEntityExtractor` won't use the generated
+features and their presence will not improve entity recognition for
+these extractors. Currently, all intent classifiers make use of
+available regex features.
+
+:::note
+
+Regex features only define entities when used in combination with the `RegexEntityExtractor`. Otherwise they  
+don't define entities nor intents! They simply provide patterns to
+help the classifier recognize entities and related intents. Hence, you
+still need to provide intent & entity examples as part of your
+training data!
+:::
+
+### Lookup Tables
+
+Lookup tables provide a convenient way to supply a list of entity
+examples. The format is as follows:
+
+```yaml
+- lookup: <lookup table name>
+  examples: |
+  - <an entity>
+  - <another entity>
+```
+
+The name of the lookup table is subject to the same constraints as the
+name of a regex feature.
+
+When lookup tables are supplied in training data, the contents are
+combined into a large regex pattern that looks for exact matches in the
+training examples. These regexes match over multiple tokens, so
+`lettuce wrap` would match `get me a lettuce wrap ASAP` as
+`[0 0 0 1 1 0]`. 
+
+These regexes are processed identically to the regular
+regex patterns directly specified in the training data and can be used
+either with the [RegexFeaturizer](../nlu/components#regexfeaturizer)
+or with the [RegexEntityExtractor](../nlu/components#regexentityextractor).
+
+**note**
+
+If you are using lookup tables in combination with the RegexFeaturizer, there must be a few examples of matches  
+in your training data. Otherwise the model will not learn to use the
+lookup table match features.
+
+**warning**
+
+You have to be careful when you add data to the lookup table.  
+For example, if there are false positives or other noise in the table,
+this can hurt performance. So make sure your lookup tables contain
+clean data.
+
+
+## Stories
```

Comment:
should we rather call this `Conversation training data (stories)`?

Reply:
Would stories & rules both fall under `Conversation training data`?

Reply:
mhm that is a good point. Maybe it should be something along the lines of 

## Conversation training data
... You can do stories that use machine learning to identify patterns ... You can use rules to implement deterministic outcomes ... (prob needs better phrasing, but just for illustration)
### Stories
...
### Rules
...

Reply:
lmk if it makes sense now